### PR TITLE
#22 - Added Versions class to represent package versions enumeration

### PR DESCRIPTION
**Pull request by olegmoz**
Issue: #22 
Added Versions class to represent package versions enumeration. Versions are going to be stored in ASTO storage

### CODE REVIEW

**Comment by 0crat** [comment]
This pull request [#25](https://github.com/artipie/nuget-adapter/issues/25) is assigned to @fanifieiev[/z](https://www.0crat.com/u/fanifieiev), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/b7577a24-a8ee-4d03-b361-b6b170719ada); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

**Comment by olegmoz** [comment]
@g4s8 the reviwer here is inactive for 3 days, could we make something about it?

**Comment by g4s8** [comment]
@0crat assign me

File: src/main/java/com/artpie/nuget/Versions.java [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+     * Packages registry content.
+     */
+    private final ByteSource content;
+
+    /**
+     * Ctor.
+     *
+     * @param content Packages registry content.
+     */
+    public Versions(final ByteSource content) {
```
Comment (g4s8) [inline src/main/java/com/artpie/nuget/Versions.java]: @olegmoz you'll receive `Flow.Publisher<ByteBuffer>` from artipie-web-server, how are you going to convert it to `ByteSource` here? Maybe you can save it without using `BlockingStorage` but with `Storage`?

Comment (olegmoz) [reply src/main/java/com/artpie/nuget/Versions.java]: @g4s8 NuGet adapter is all non-reactive now and it's entry point (Repository) uses BlockingStorage. I am planning to change it later and added an issue for it https://github.com/artipie/nuget-adapter/issues/5

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/artipie/nuget-adapter/issues/25#issuecomment-586878063))

@g4s8 This pull request [#25](https://github.com/artipie/nuget-adapter/issues/25) is assigned to @g4s8[/z](https://www.0crat.com/u/g4s8), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/de2ea318-6276-49f3-82c3-534bbc22890e); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be no monetary reward for this job

**Comment by 0crat** [comment]
Manual assignment of issues is discouraged, see [§19](http://www.zerocracy.com/policy.html#19): -5 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Job was finished in 0 hours, bonus for fast delivery is possible (see [§36](http://www.zerocracy.com/policy.html#36))

**Comment by 0crat** [comment]
Order was finished: +20 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Pull request merged**